mozetl/cli.py

```diff
@@ -10,3 +10,4 @@ def entry_point():
 
 entry_point.add_command(churn.main, "churn")
 entry_point.add_command(dashboard.main, "search_dashboard")
+entry_point.add_command(search_rollups.main, "search_rollup")
```

Comment:
why plural in the first instance, singular in the second instance?

Reply:
Somewhere along the line, I forgot to grammar. The airflow job calls it `search_rollup` so this definitely calls for more consistency. The name gets used in as in "running the daily search rollup" vs "running the search rollups". I'll attach it to an upcoming patch to remove the manifest files and store the data in parquet.